gt/main.c

```diff
@@ -138,6 +146,9 @@ gt_parse_incoming_pkt(struct rte_mbuf *pkt, struct gt_packet_headers *info)
 		info->inner_ip_ver = ETHER_TYPE_IPv6;
 		info->l4_proto = inner_ipv6_hdr->proto;
 		info->l4_hdr = &inner_ipv6_hdr[1];
+		inner_ipv6_hdr->vtc_flow =
+			(inner_ipv6_hdr->vtc_flow & (~IPTOS_ECN_MASK << 20)) |
```

Comment:
Shouldn't the expression `(~IPTOS_ECN_MASK << 20)` really be `~(IPTOS_ECN_MASK << 20)`?